docs/developer-docs/functionality/sns/sns.md

```diff
@@ -1,2 +1,230 @@
+   to another deployment are provided, thoroughly tested, and _blessed_ by the IC
+   community (through NNS proposals). All the SNS communities have to do is to vote
+   to upgrade the SNS according to the blessed upgrade path. This will automatically
+   fetch the right canister versions that have previously been "blessed" and
+   upgrade the SNS canisters to them. 
+   SNS communities can nevertheless customize their SNS by choosing a variety
+   of parameters.
+   Such SNSs are hosted on an _SNS subnet_ that exclusively hosts blessed SNSs.
+   This simplifies the verification for end users who can simply verify that an SNS
+   is running on the SNS subnet and infer that the canisters run blessed versions
+   rather than possibly altered code.
+   
+2) **_Self-deploy_ an SNS and manually upgrade it.**
+   Any SNS community can choose to deploy the SNS code, which is open source, on an 
+   application subnet. They can then choose to follow the blessed upgrade path or deviate 
+   from this path, e.g., leaving out some versions, or even modify the canisters’ code 
+   in a completely different way. In this option, canister upgrades require more 
+   actions of the SNS community (i.e., compiling new wasms and make upgrade proposals)
+   and the SNS community has to ensure that the upgrades are secure. This includes 
+   ensuring that alternative canister versions are compatible and that upgrades to 
+   newer versions do not break any functionality. 
+   In order to provide better security for such SNSs and other security-sensitive
+   canisters, an application subnet with higher replication will soon be available.
+
+3) **Build your own DAO or use frameworks provided by others to build your DAO.**
+   While this is conceptually similar to the second option, we would like to emphasize
+   that there are of course other design than the SNS that also build a DAO.
+   For most of these, the implications for the communities will be similar to the
+   second option: The DAO communities will have to maintain the DAO versions, or trust
+   a third party to do so and such DAOs can be deployed on a higher-replication 
+   application subnet that will soon be available. 
+   
+
+These possibilities allow communities to choose between using DAOs that are provided
+as a service by the IC, where maintenance is as automated as possible, and DAOs
+that have full flexibility of how they can evolve. Because the possibilities for
+Options 2 and 3 are unbounded, we focus here on explaining the SNS in Option 1 in
+more detail.
+
+## SNS canisters
+The SNS consists of a governance canister, a ledger canister, 
+a root canister, and a decentralization sale canister that is explained in the next
+section.
+
+The _ledger canister_ implements the 
+[ICRC-1 standard](https://github.com/dfinity/ICRC-1)
+and contains SNS tokens, which are unique tokens for each SNS.
+It stores which accounts own how many SNS tokens and the history of transactions 
```

Comment:
```suggestion
It stores which accounts owns how many SNS tokens and the history of transactions 
```

Reply:
I don't agree with this as "accounts" is plural and will ignore it.
please let me know if I overlook something and am wrong :)